### PR TITLE
tikv: fix code coverage script

### PR DESCRIPTION
**Pull request by glorv**
After this change we still need to wait tikv to update the rust toolchain to a newer version before this script can succeed due to https://github.com/rust-lang/rust/issues/118850

### CODE REVIEW

File: jenkins/pipelines/ci/tikv/tikv_ghpr_coverage.groovy [hunk 1]
```diff
@@ -64,12 +64,12 @@ stage("Cover") {
                 export FAIL_POINT=1
                 export ROCKSDB_SYS_SSE=1
                 export CARGO_INCREMENTAL=0
-                export RUSTFLAGS="-Zinstrument-coverage"
+                export RUSTFLAGS="-C instrument-coverage"
```
Comment (Connor1996) [inline jenkins/pipelines/ci/tikv/tikv_ghpr_coverage.groovy:67]: What's the difference

Comment (glorv) [reply jenkins/pipelines/ci/tikv/tikv_ghpr_coverage.groovy:67]: -Zinstrument-coverage is not supported anymore

**Review by Connor1996 — APPROVED** [review]
LGTM

**Review by purelind — APPROVED** [review]
/lgtm

**Pull request merged**